### DISCUSSION TITLE
Questions sources

### DISCUSSION THREAD

**Discussion by hazzuk** [Maerciez/TheMillionaireGame#86] [Guides]
As I'll be trying to host this game soon, I spent an hour or so looking for places to find questions. Thought it might be useful to share what I found with others here.

> [!NOTE] 
> If you have your own sources for questions, and possibly in different languages, leave a comment below.

<br>

| Source | Lang | Size | Format | Region |
|---|---|---|---|---|
| [millionaireDB.com](https://www.millionairedb.com/questions/pages/1/) | en | 3957 | website | mixed? |
| [Millionaire-Questions.xlsx](https://docs.google.com/spreadsheets/d/16RWCnvLcyQKH8QEqSUlenf8_YNjIuTxqNssCvZF-URA/) | en | 6k | .xlsx | usa? |
| [uk.wwbm.com](https://uk.wwbm.com/) | en | ? | web game | uk |
| [opentdb.com](https://opentdb.com/api_config.php) | en | 4148 | web/api | mixed |
| [aaronnech/Who-Wants-to-Be-a-Millionaire](https://github.com/aaronnech/Who-Wants-to-Be-a-Millionaire) | en | 200~ | .json | india? |
| [lukewlodarczyk/Who-Wants-to-Be-a-Millionaire](https://lukewlodarczyk.github.io/Who-Wants-to-Be-a-Millionaire/#/game) | en | ? | web game | uk? |
| [wwtbambored.com](https://www.wwtbambored.com/viewforum.php?f=3) | en | Large | transcripts | usa |
| [iamemc/who-wants-to-be-a-millionaire](https://github.com/iamemc/who-wants-to-be-a-millionaire/blob/main/questionsDatabase.py) | en | 150 | .py | usa? |
| [quiz-questions.uk](https://quiz-questions.uk/who-wants-to-be-a-millionaire/) | en | 150 | website | uk |
| [bahadiri/Millionaire](https://github.com/bahadiri/Millionaire) | tr | 1908 | .csv/sql | mixed |

**Comment by Macronair** [comment]
Wow! I didn't know that these websites existed. Only opentdb sounds familiar to me.
Thanks for your input! 😄
